src/main/java/com/quorum/gauge/services/GraphQLService.java

```diff
@@ -0,0 +1,75 @@
+package com.quorum.gauge.services;
+
+import com.quorum.gauge.common.QuorumNetworkProperty;
+import com.quorum.gauge.common.QuorumNode;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+@Service
+public class GraphQLService extends AbstractService {
```

Comment:
Methods in service class should return `Observable` instead of explicit

Reply:
Added Observable wrapper to the primitive return type. Please check if this is the correct practice.

Reply:
Changed it to use `Single` instead of `Observable` as there are no multiple items to be emitted in each GraphQL query. Feel like this is a cleaner way.